index.html

```diff
@@ -1,23 +1,63 @@
-<html>
-  <head>
-        <title>Hey Let's Add Some Numbers!</title>
-        <script language="javascript">
-                function addNumbers()
-                {
-                        var val1 = parseInt(document.getElementById("value1").value);
-                        var val2 = parseInt(document.getElementById("value2").value);
-                        var ansD = document.getElementById("answer");
-                        ansD.value = val1 + val2;
-                }
-        </script>
-  </head>
-  <body>
-        Gimmie a Number  <input type="text" id="value1" name="value1" value=""/>
-        Gimmie Another Number and I'll Add Them Together For You!  <input type="text" id="value2" name="value2" value=""/>
-        <input type="button" name="Sumbit" value="CLICK THIS BUTTON FOR THE ANSWER!" onclick="javascript:addNumbers()"/>
-        The Answer is.. <input type="text" id="answer" name="answer" value=""/>
+<body>  
+ 
+      <br/>
+  Gimmie Some Values!<input id="fn" type="text" onblur="myFunction()" />
+    <select id="ddlselect" onchange="myFunction()">
+    <option value="Select">Pick a Function</option>
+          <option value="+">+</option>
+          <option value="-">-</option>
+          <option value="*">*</option>
+          <option value="/">/</option>
+         
+      </select>   
+      <input id="sn" type="text" onblur="myFunction()" />  <label>=</label>
+      <input id="rn" type="text" readonly/>
+      <script type="text/javascript">
+         
+          function myFunction()
+          {
+              
+              var x =  document.getElementById("ddlselect").value;
+              var fn = parseInt(document.getElementById("fn").value);
```

Comment:
We have learned how to make separate files as well it will be much easier later for you to know how to seperate html, javascript and css into different files and import them.